troubleshoot/observability/troubleshoot-your-universal-profiling-agent-deployment/troubleshoot-universal-profiling-backend.md

```diff
@@ -2,6 +2,10 @@
 navigation_title: "Troubleshoot the backend"
 mapped_pages:
   - https://www.elastic.co/guide/en/observability/current/profiling-self-managed-troubleshooting.html
+applies_to:
+  stack: all
+  serverless:
+    observability: all
```

Comment:
Depending on the answer to the question in `troubleshoot/observability/troubleshoot-your-universal-profiling-agent-deployment.md`, this might need to be removed.

Reply:
```suggestion
```

Reply:
That's correct. It's not currently in Serverless.